src/backend/distributed/commands/alter_table.c

```diff
@@ -1645,6 +1748,15 @@ ReplaceTable(Oid sourceId, Oid targetId, List *justBeforeDropCommands,
 								quote_qualified_identifier(schemaName, sourceName))));
 	}
 
+	/*
+	 * we need to prepare rename identities commands before dropping the original table
+	 */
```

Comment:
would be good to add a comment on why we need different commands

Reply:
@gokhangulbiz I guess here we are looking for a comment like this, this comment can either be here or as a description of `PrepareRenameCommands` function:
``` C
/*
 * We need to prepare rename identities commands before dropping the original table,
 * otherwise we can't find the original names of the identity sequences.
 * We prepare separate commands for the coordinator and the workers because:
 * In the coordinator, we simply need to rename the identity sequences to their names on the old table, because right now the identity sequences have default names generated by Postgres with the creation of the new table
 * In the workers, we have not dropped the original identity sequences, so what we do is we alter the columns and set their default to the original identity sequences, and after that we drop the new sequences.
*/
```